aws_lambda_builders/workflows/go_modules/workflow.py

```diff
@@ -0,0 +1,45 @@
+"""
+Go Modules Workflow
+"""
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from aws_lambda_builders.actions import CopySourceAction
+
+from .actions import GoModulesBuildAction
+from .builder import GoModulesBuilder
+from .utils import OSUtils
+
+
+class GoModulesWorkflow(BaseWorkflow):
+
+    NAME = "GoModulesBuilder"
+
+    CAPABILITY = Capability(language="go",
+                            dependency_manager="modules",
+                            application_framework=None)
+
+    def __init__(self,
+                 source_dir,
+                 artifacts_dir,
+                 scratch_dir,
+                 manifest_path,
+                 runtime=None,
+                 osutils=None,
+                 **kwargs):
+
+        super(GoModulesWorkflow, self).__init__(
+            source_dir,
+            artifacts_dir,
+            scratch_dir,
+            manifest_path,
+            runtime=runtime,
+            **kwargs)
+
+        if osutils is None:
+            osutils = OSUtils()
+
+        executable_name = osutils.basename(source_dir)
+        builder = GoModulesBuilder(osutils)
+        self.actions = [
+            GoModulesBuildAction(source_dir, artifacts_dir, executable_name, builder),
+            CopySourceAction(source_dir, artifacts_dir, only=[executable_name]),
```

Comment:
Is this needed? `go build` can take an output dir. Why not do that as well here? 

Reference: https://github.com/awslabs/aws-lambda-builders/pull/54/files#diff-c11c930189e8062aced3a130ff5ff0efR55

Reply:
It's not, the first call is actually placing things in the right place, so I don't need to `CopySourceAction` call. Fixing.

Reply:
Fixed.